packages/account/src/Components/terms-of-use/terms-of-use.tsx

```diff
@@ -132,12 +137,30 @@ const TermsOfUse = ({
                                                 />
                                             }
                                         />
+                                        <Hr />
```

Comment:
a doubt, is `Hr` suppose to stay with the below `Field`? if so, this should come after `is_spain_residence &&` condition, right?